src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Extensions/RoslynLspExtensions_Range.cs

```diff
@@ -12,4 +14,11 @@ public static LinePositionSpan ToLinePositionSpan(this Range range)
 
     public static string ToDisplayString(this Range range)
         => $"{range.Start.ToDisplayString()}-{range.End.ToDisplayString()}";
+
+    public static VsLspRange ToVsLspRange(this Range range)
```

Comment:
Please don't add this extension method. It is 100% intentional that the `RoslynLspExtensions` and `VsLspExtensions` never cross paths. There is `VsLspFactory` class that can be used at calls sites instead. Just convert your Roslyn LSP Range to a LinePositionSpan and call `VsLspFactory.CreateRange(LinePositionSpan)`.